docs/hosted-orchestration/running-tests.md

```diff
@@ -0,0 +1,202 @@
+---
+id: running-tests
+title: Hosted Orchestration Running Tests
+sidebar_label: Running Tests
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+This page outlines how to run your browser and mobile tests in Hosted Test Orchestration.
+
+## What You'll Need
+
+- A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up)).
+- Your Sauce Labs [Username and Access Key](https://app.saucelabs.com/user-settings).
+- The SauceCTL client installed. Read [Using the saucectl CLI](../dev/cli/saucectl) to learn more.
+- A Docker image containing your tests. Read [Building Images](./building-images) if you need help.
+
+## Starting an Execution
+
+You can interact with Hosted Orchestration through the SauceCtl CLI. An example SauceCTL configuration is below.
+
+<Tabs
+     defaultValue="SauceCTL"
+     values={[
+       {label: 'SauceCTL', value: 'SauceCTL'},
+     ]}>
+  <TabItem value="SauceCTL">
+
+  ```yaml
+    apiVersion: v1alpha
+    kind: htexec
+    sauce:
+      region: us-west-1
+    suites:
+      - name: run sauce test
+        image: saucelabs/sl-demo-docker-primary:0.0.1
+        imagePullAuth:
+          user: $SAUCE_IMAGE_USER
+          token: $SAUCE_IMAGE_TOKEN
+        entrypoint: "mvn test"
+        files:
+          - src: "runsauce.json"
+            dst: "/workdir/runsauce.json"
+        artifacts:
+          - "/path/inside/container/file.log"
+        env: 
+          KEY: value
+  ```
+
+  Then run with
+
+  ```bash
+    saucectl run
+  ```
+
+  You should receive a successful output that looks like
```

Comment:
Since it looks like we're not going to ship with image streaming, we should give the user some kind of indication that the `saucectl` job may take a very a long time and that they'll have to wait.